pkgs/os-specific/linux/nsncd/default.nix

```diff
@@ -2,21 +2,27 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, nix-gitignore
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "nsncd";
-  version = "unstable-2024-03-18";
+  version = "unstable-2024-08-06";
 
   src = fetchFromGitHub {
     owner = "twosigma";
     repo = "nsncd";
-    rev =  "7605e330d5a313a8656e6fcaf1c10cd6b5cdd427";
+    rev = "7605e330d5a313a8656e6fcaf1c10cd6b5cdd427";
     hash = "sha256-Bd7qE9MP5coBCkr70TdoJfwYhQpdrn/zmN4KoARcaMI=";
   };
 
-  cargoHash = "sha256-i1rmc5wxtc631hZy2oM4d6r7od0w8GrG7+/pdM6Gqco=";
+  cargoHash = "sha256-06fbzx+rqO4k1Cm/UaLk5SRs1wWPSpOG/qCie4c/nPQ=";
+
+  # TOREMOVE when https://github.com/twosigma/nsncd/pull/119 gets merged.
+  cargoPatches = [
+    ./0001-cargo-bump.patch
+    ./0002-ignored-unused-hostent-field.patch
```

Comment:
Might be nicer to avoid a vendored patch with

```nix
RUSTFLAGS = "-A dead_code";
```

Reply:
done